lldb/source/Expression/IRExecutionUnit.cpp

```diff
@@ -336,6 +339,30 @@ void IRExecutionUnit::GetRunnableInfo(Status &error, lldb::addr_t &func_addr,
     m_execution_engine_up->setObjectCache(m_object_cache_up.get());
   }
 
+  // Record the size of the sections discounting the extra space used 
+  // by the JIT. This is a hack and should be removed when we switch
+  // from MCJIT to ORCJIT.
```

Comment:
///

Reply:
This is declared in the middle of a function, do doxygen comments work here?